.github/workflows/release.yml

```diff
@@ -26,14 +26,16 @@ jobs:
         run: |
           export PATH=${PATH}:`go env GOPATH`/bin
           go get -v -t -d ./...
-      - name: Install gox
+      - name: Build govvv
         run: |
-          export PATH=${PATH}:`go env GOPATH`/bin
-          go get github.com/mitchellh/gox  
+          go build -modfile .bingo/govvv.mod -o=. github.com/ahmetb/govvv
+      - name: Build gox
+        run: |
+          go build -modfile .bingo/gox.mod -o=. github.com/mitchellh/gox
```

Comment:
need to verify that this wouldn't mess with the `buck` build below that doesn't need govvv yet

Reply:
@asutula seems okay right?

Reply:
Yea, should be fine. Just building the `govvv` executable shouldn't cause any problems.